apps/spruce/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import useTablePagination from "hooks/useTablePagination";
+import { CustomMeta, CustomStoryObj } from "test_utils/types";
+import Pagination from ".";
+
+export default {
+  component: Pagination,
+} satisfies CustomMeta<typeof Pagination>;
+
+const DefaultImpl = (args: React.ComponentProps<typeof Pagination>) => {
+  const { page } = useTablePagination();
+
+  return (
+    <>
+      <p>Query Params: {page}</p>
```

Comment:
[nit] I would say this is kind of redundant since it doesn't actually reflect the query params, just the same value that is passed to the Pagination component

Reply:
I updated the verbiage to reflect the usage instead. lmk if that works or if you think I should remove it entirely!